remove .data from torch/nn doc

albanD opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #31482 remove simple .data from torch/nn
* **#31481 remove .data from torch/nn doc**
* #31480 remove .data from torch/jit
* #31479 Remove some .data and clearly mark why we keep some

Differential Revision: [D19303242](https://our.internmc.facebook.com/intern/diff/D19303242)

albanD committed:
remove .data from torch/nn doc

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit bd4e70e:


* **2/2** broken upstream at merge base fc3103b (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=fc3103b1162a3312e4ad74005d0b698a333c0750&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **0/2** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=bd4e70e6e282f3a1cd3c646259e8e72e2c7ac1bc).

### :construction: 2 upstream failures recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=77626599)
* [pytorch_linux_xenial_py3_6_gcc5_4_test](https://dr.pytorch.org/build-details.html?build_id=77626730)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

albanD committed:
Update on "remove .data from torch/nn doc"




[ghstack-poisoned]

albanD committed:
Update on "remove .data from torch/nn doc"




[ghstack-poisoned]

albanD committed:
Update on "remove .data from torch/nn doc"


Differential Revision: [D19303242](https://our.internmc.facebook.com/intern/diff/D19303242)

[ghstack-poisoned]
